docs/player-support/snapcast.md

```diff
@@ -1,9 +1,9 @@
 # Snapcast ![Preview image](../assets/icons/snapcast-icon.svg){ width=70 align=right }
 
 Music Assistant supports Snapcast, a powerful solution for synchronized multi-room audio streaming. Snapcast enables seamless playback across various devices, creating an immersive audio experience.
-Whether you're using Snapcast-compatible speakers or devices like the Raspberry Pi, you can enjoy synchronized audio playback effortlessly. This component was contributed and maintained by [SantiagoSotoC](https://github.com/Santiagosotoc)
+Whether you're using Snapcast-compatible speakers or devices like the Raspberry Pi, you can enjoy synchronized audio playback effortlessly. This component is contributed and maintained by [SantiagoSotoC](https://github.com/Santiagosotoc)
 
-In order to use this provider you need to also have a Snapcast server running on your network. The diagram below shows a possible combination of outputs. In the diagram a Raspberry Pi runs the server which communicates to MA and all of the clients. The server running Pi is also running Snapclient and is connected to a set of speakers. Then there is another Pi running Snapclient in another room, a phone running Snapdroid and a laptop running Snapweb,
+In order to use this provider you need to also have a Snapcast server running on your network (a server is included with this provider). The diagram below shows a possible combination of outputs. In the diagram a Raspberry Pi runs the server which communicates to MA and all of the clients. The server running Pi is also running Snapclient and is connected to a set of speakers. Then there is another Pi running Snapclient in another room, a phone running Snapdroid and a laptop running Snapweb,
```

Comment:
I would rephrase this as I expect most people will just use the builtin server. So rewrite it to the fact that music assistant includes a snapserver but if you already have an existing snapserver running on your network, there's an option to use that one instead.